server/validators/participant.ts

```diff
@@ -97,13 +99,19 @@ export const ParticipantSchema = yup
   .object()
   .noUnknown('Unknown field in form')
   .shape({
+    program: yup.string().oneOf(programs, 'Program should be HCA or MHAW'),
+
     // Eligibility
     eligibility: yup
       .boolean()
       .typeError(errorMessage)
       .required(errorMessage)
       .test('is-true', errorMessage, (v) => v === true),
 
+    educationalRequirements: yup
+      .string()
+      .oneOf(yesOrNo, 'Educational requirements should be yes or no'),
```

Comment:
capitalize yes and no to match other messages?

Reply:
I do think we have to distinct between "I don't know" (FE) and "unknown" (what we backfill existing participants with). So I do think we change this validation